### DISCUSSION TITLE
No releases in a long time

### DISCUSSION THREAD

**Discussion by Alexdat2000** [zxing-cpp/zxing-cpp#875] [General]
Hello! Are there any plans to make new releases? It has been over a year since last release and a new binarizing algorithm was added in that time. We want to use a stable version of the library, but they are not up-to-date
Thanks in advance!

**Comment by axxel** [comment]
You absolutely right. I thought I'd make at least the 1 year mark since the last release but life just keeps me very busy these days. I'll 'promise' to make a release next week when I have some free time on my hands.

**Reply by Alexdat2000** [reply]
Thank you so much!

**Reply by axxel** [reply]
I "published" a 2.3.0-rc1 first, mainly because I never tried to do a pre-release. I don't expect much reaction today and will likely do the release tomorrow, because 1.1.25 sounds nice ;)

**Comment by axxel** [comment]
While trying to decide which is the best zint version to download in case the "bundled zint" is not to be used within zxing-cpp, I tried the latest release (2.13.0) but noticed that there is a major feature missing: `BARCODE_MEMORY_FILE`. I then noticed that zint didn't have a release for over a year, either. @gitlost do you have any immediate release plans for a 2.14.0 release?

Distributions like Debian will certainly want to dynamically link against their own libzint library, which will then only work, once there is a sufficiently capable one. But this will only really be of relevance, if they wanted to enable the "experimental api", which I would not recommend for zxing-cpp v2.3.0.

**Comment by axxel** [comment]
Technically, I didn't quite keep my promise, but here you go: https://github.com/zxing-cpp/zxing-cpp/releases/tag/v2.3.0
